files/startup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+jinja2 server.xml.j2 > tomcat/conf/server.xml
+
+exec /app/tomcat/bin/catalina.sh "$@"
```

Comment:
@fabricetriboix It looks like  `VirtualViewerJavaHTML5.xml`  file is not added for parameterization.

Reply:
> @fabricetriboix It looks like `VirtualViewerJavaHTML5.xml` file is not added for parameterization.

@naveen-armedia I don't understand your comment. Could you please explain in more details what you mean? Thanks

Reply:
@fabricetriboix As far as I remember In the last call when we are with Jhon Shinnick we were facing an issue to integrate the snowbound with Arkcase-CE. The snowbound pod was returning a 404 error. And you and Jhon dig deep into the root cause and it was found out to be an issue with `VirtualViewerJavaHTML5.xml` file where `snowbound_host` and `arkcase_host_name` variables are not been handled. Please find below the content of `VirtualViewerJavaHTML5.xml` config file. Without having these variables expanded to proper values I don't think we can have snowbound work properly.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<Context>
    <!-- These parameters are used in web.xml (their replacement happens in ArkCaseServletContextListener.setInitParameters()) -->
    <Parameter name="snowbound.base" value="https://{{ snowbound_host }}" override="false"/>
    <Parameter name="arkcase.base" value="https://{{ arkcase_host_name }}:8843" override="false"/>
</Context>
```

Reply:
@naveen-armedia This file is meant to stay this way inside the container. These are, I believe, Java system variables that get expanded at runtime by Snowbound. Bojan did some magic to achieve this and I don't know the details of how it works. If you want to have more details, please contact Bojan.

Reply:
@fabricetriboix If you are not aware could you please check with David Miller or Dimitar or John Shinnick regarding these variables? As far as I know, `snowbound_host` and `arkcase_host_name` are the ansible facts that are available in the `arkcase-ce-facts.yml` file.

Reply:
There variables will need to be set in the CATALINA_OPTS, which will be done by the Snowbound Helm chart. I am not sure what you want me to do here?

Reply:
Alright okay. Then please change the format something like `{{ snowbound_host }}` to ${snowbound.host} and `{{arkcase_host_name}}` to ${arkcase.hostname} so that these variables will be expanded from CATALINA_OPTS. 

Please refer to the snowbound ansible role.

Reply:
I don't understand... The `VirtualViewerJavaHTML5.xml` already has `${snowbound_host_port}` and `${arkcase_host_port}`. What do you want me to do?

Reply:
oh right. Thanks, Fabrice I assumed you have not changed it.